apps/print/frontend/src/screens/machine_locked_screen.tsx

```diff
@@ -1,27 +1,46 @@
-import { H1, InsertCardImage, Main, Screen } from '@votingworks/ui';
-import React from 'react';
-import { getElectionRecord } from '../api';
+import styled from 'styled-components';
+import { ElectionInfoBar, H1, H3, Main, Screen } from '@votingworks/ui';
+import { getElectionRecord, getMachineConfig } from '../api';
+import { Column } from '../layout';
+
+const LockedImage = styled.img`
+  margin-right: auto;
+  margin-bottom: 1.25em;
+  margin-left: auto;
+  height: 20vw;
+`;
 
 export function MachineLockedScreen(): JSX.Element | null {
   const getElectionRecordQuery = getElectionRecord.useQuery();
+  const getMachineConfigQuery = getMachineConfig.useQuery();
 
-  if (!getElectionRecordQuery.isSuccess) {
+  if (!getElectionRecordQuery.isSuccess || !getMachineConfigQuery.isSuccess) {
     return null;
   }
 
-  const electionRecord = getElectionRecordQuery.data;
+  const electionDefinition = getElectionRecordQuery.data?.electionDefinition;
+  const electionPackageHash = getElectionRecordQuery.data?.electionPackageHash;
+  const machineConfig = getMachineConfigQuery.data;
   return (
     <Screen>
       <Main centerChild>
-        <React.Fragment>
-          <InsertCardImage cardInsertionDirection="right" />
-          <H1 align="center" style={{ maxWidth: '50rem' }}>
-            {electionRecord
-              ? 'Insert a card to unlock'
-              : 'Insert an election manager card to configure VxPrint'}
-          </H1>
-        </React.Fragment>
+        <Column style={{ alignItems: 'center' }}>
+          <LockedImage src="/locked.svg" alt="Locked Icon" />
+          <H1 style={{ marginTop: '0' }}>VxPrint Locked</H1>
+          <H3 style={{ fontWeight: 'normal' }}>
+            {electionDefinition
+              ? 'Insert card to unlock.'
+              : 'Insert an election manager card to configure VxPrint.'}
```

Comment:
nit: no period for consistency with other apps :/

Reply:
<img width="916" height="565" alt="Screenshot 2025-12-04 at 2 30 39 PM" src="https://github.com/user-attachments/assets/92e8e6dc-4331-415d-82c3-6f01106a9634" />

Admin has period :/